_posts/2017-04-04-things-the-rust-community-team-does.md

```diff
@@ -0,0 +1,101 @@
+
+Still, where there's no information, misconceptions arise, in all directions: either that the team doesn't do anything at all or that the teams size and work is way exaggerated.
+
+So, I recently accidentally wrote a pretty exhausive list, which I'd like to extend and republish here. A lot of what we do is all over the place and in a DIY fashion, so we learn a lot of skills on the go. If you have experience with any of these - please get involved!
+
+## Tend a calendar
+
+Perhaps the oldest piece of community documentation is the [Rust community calendar](https://calendar.google.com/calendar/embed?src=apd9vmbc22egenmtu5l6c5jbfc@group.calendar.google.com&pli=1), created by Erick and maintained to this date. Ping us to get access. We regularly check if recurring events are still on.
+
+## Watch the Forums
+
+We read and engage in the comment sections. First of all, we believe that the project should be engaged into discussions about it. Hat tip to Sri Ramkrishna for giving a whole talk about that subject: [Swimming with chum in shark infested water](https://www.youtube.com/watch?v=VYrL_l984-E). This engagement must be complex and assertive.
+
+Second, it is a great source of information about our project. We take appropriate action by either documenting these issues for the project or by getting people into a discussion with the people that are an expert in their subject.
+
+## Find Creative Solutions
+
+We come up with creative solutions to worldwide community issues, including marketing, funding, growth. (*shiver*) The interesting thing about doing these in a FOSS project is that they need to low- or zero-cost and time effective, which means many standard solutions don't apply.
+
+Marketing happens often in the sense that we think about how to get meaningful information out to people. A good example is that if you find us at a conference, you might also find one of edunhams handy [postcard with important links](https://github.com/rust-community/resources/blob/gh-pages/rust_printed-card.pdf) next to it.
+
+## General Promotion
+
+We help Rust projects at promotion. A recent example is [rust-bios survey](https://github.com/rust-bio/rust-bio/issues/119#issuecomment-290196421) for a grant application. Instead of having individuals promote themselves, we use our experience to help out, without requiring a formal request.
+
+## Meeting Organisation
+
+This isn't always done by the community team, but we do make sure that people can have a meeting with the core or other teams to talk about specific subjects. This mostly involves both finding interesting people to speak to, but also encouraging community members that it is definitely okay to try to get some of the other teams time.
+
+Standards are the Rust production user meetings, but there have also been meetings on other subjects, like graphics or embedded systems.
+
+## Identify Upcoming Issues
```

Comment:
I think you need to introduce this section a little more. How about something like:

One of our important roles is to listen in on our worldwide community, and pass that feedback on to the rest of the teams about the state of the project. We do this through a number of different avenues, like our annual community and commercial survey...